ant/tools/load-aos-ant.cjs

```diff
@@ -0,0 +1,96 @@
+const AoLoader = require('@permaweb/ao-loader');
+const fs = require('fs');
+const path = require('path');
+
+const address = ''.padEnd(43, 'a');
+/* ao READ-ONLY Env Variables */
+const env = {
+  Process: {
+    Id: ''.padEnd(43, '1'),
+    Owner: address,
+    Tags: [{ name: 'Authority', value: 'XXXXXX' }],
+  },
+  Module: {
+    Id: ''.padEnd(43, '1'),
+    Tags: [{ name: 'Authority', value: 'YYYYYY' }],
+  },
+};
+
+async function main() {
+  const wasmBinary = fs.readFileSync(
+    path.join(
+      __dirname,
+      'fixtures/aos-9afQ1PLf2mrshqCTZEzzJTR2gWaC9zNPnYgYEqg1Pt4.wasm',
+    ),
+  );
+  // read the lua code from the dist
+  const luaCode = fs.readFileSync(
+    path.join(__dirname, '../dist/aos-ant-bundled.lua'),
+    'utf-8',
+  );
+  // for initializing ant state
+  const initState = JSON.stringify({
+    balances: { [address]: 1 },
+    controllers: [address],
+    name: 'ANT-ARDRIVE',
+    owner: address,
+    records: {
+      '@': {
+        transactionId: 'UyC5P5qKPZaltMmmZAWdakhlDXsBF6qmyrbWYFchRTk',
+        ttlSeconds: 3600,
+      },
+    },
+    ticker: 'ANT',
+  });
+  // Create the handle function that executes the Wasm
+  const options = {
+    format: 'wasm32-unknown-emscripten',
+    inputEncoding: 'JSON-1',
+    outputEncoding: 'JSON-1',
+    memoryLimit: '524288000', // in bytes
+    computeLimit: (9e12).toString(),
+    extensions: [],
+  };
+
+  const testCases = [
+    ['Eval', { Module: ''.padEnd(43, '1') }, luaCode],
+    ['Initialize-State', {}, initState],
+    ['Info', {}],
+    ['Get-Records', {}],
+    ['Transfer', { Recipient: 'iKryOeZQMONi2965nKz528htMMN_sBcjlhc-VncoRjA' }],
```

Comment:
can we add expectations here and use `asserts` instead of console.logs so we can use in CI/CD?